middleware.ts

```diff
@@ -1,18 +1,22 @@
 import { NextResponse, NextRequest } from 'next/server';
 
 // TODO: These routes will need to be updated.
-const protectedPaths = ['/dmps/', '/admin/']
+const protectedPaths = ['/dmps/', '/admin/', '/template']
```

Comment:
It may make more sense to define which paths are not protected. There will be fewer public facing pages than protected pages.

Reply:
no need to modify now, just pointing it out